### DISCUSSION TITLE
glossary of the benchmarks project

### DISCUSSION THREAD

**Discussion by pvgenuchten** [soilwise-he/soil-vocabs#17] [Ideas]
tThe benchmarks project maintains a glossary at https://soilhealthbenchmarks.eu/glossary

The glossary has adoped some concepts of the [landmark glossary](https://landmarkproject.eu/landmark-glossary/) 

The glossary is not available in a skos model, it is maintained in a wordpress instance

**Comment by pvgenuchten** [comment]
the benchmarks glossary has these terms, a link to similar term in agrovoc, gemet and wikidata is added,  where available

| category | concept | uri | agrovoc | EEA | Wikidata |
| --- | --- | --- | --- | --- | --- |
| ECOSYSTEM SERVICES |  | https://soilhealthbenchmarks.eu/glossary/ecosystem-services | http://aims.fao.org/aos/agrovoc/c_1348040570280 | https://www.eea.europa.eu/help/glossary/chm-biodiversity/ecological-or-ecosystem-services | https://www.wikidata.org/wiki/Q295865 |
| ECOSYSTEM SERVICES | Provisioning services | https://soilhealthbenchmarks.eu/glossary/provisioning | http://aims.fao.org/aos/agrovoc/c_7e12eb3f | | |
| ECOSYSTEM SERVICES | Regulating services | https://soilhealthbenchmarks.eu/glossary/regulation-and-maintenance | http://aims.fao.org/aos/agrovoc/c_eff3625d | | |
| ECOSYSTEM SERVICES | Cultural services | https://soilhealthbenchmarks.eu/glossary/cultural | http://aims.fao.org/aos/agrovoc/c_f359b8d7 | | |
| ECOSYSTEM SERVICES | Soil ecosystem services | https://soilhealthbenchmarks.eu/glossary/soil-ecosystem-services | http://aims.fao.org/aos/agrovoc/c_0f955994 | | |
| ECOSYSTEM SERVICES | Ecosystem services | https://soilhealthbenchmarks.eu/glossary/ecosystem | http://aims.fao.org/aos/agrovoc/c_1348040570280 | | |
| ECOSYSTEM SERVICES | Natural capital | https://soilhealthbenchmarks.eu/glossary/natural-capital | http://aims.fao.org/aos/agrovoc/c_dfe84e21 | https://www.eea.europa.eu/help/glossary/eea-glossary/natural-capital | https://www.wikidata.org/wiki/Q1540268 | | |
| SOIL FUNCTIONS |  | https://soilhealthbenchmarks.eu/glossary/soil-functions | http://aims.fao.org/aos/agrovoc/c_e5629cbd | https://www.eea.europa.eu/help/glossary/gemet-environmental-thesaurus/soil-function | https://www.wikidata.org/wiki/Q7554911 |
| SOIL FUNCTIONS | Water regulation and purification | https://soilhealthbenchmarks.eu/glossary/water-regulation-and-purification |  | | |
| SOIL FUNCTIONS | Water filtering (soil) | | http://aims.fao.org/aos/agrovoc/c_25b98c3e | | |
| SOIL FUNCTIONS | Infiltration water | | http://aims.fao.org/aos/agrovoc/c_25321 | https://www.eea.europa.eu/help/glossary/gemet-environmental-thesaurus/infiltration | https://www.wikidata.org/wiki/Q1127330 |
| SOIL FUNCTIONS | Water storage | | http://aims.fao.org/aos/agrovoc/c_8327 | | https://www.wikidata.org/wiki/Q7973769 |
| SOIL FUNCTIONS | Carbon sequestration and climate regulation | https://soilhealthbenchmarks.eu/glossary/carbon-sequestration-and-climate-regulation | | | |
| SOIL FUNCTIONS | Carbon sequestration | | http://aims.fao.org/aos/agrovoc/c_331583 | https://www.eea.europa.eu/help/glossary/eea-glossary/carbon-sequestration | https://www.wikidata.org/wiki/Q15305550 |
| SOIL FUNCTIONS | Nutrient cycling | https://soilhealthbenchmarks.eu/glossary/nutrient-cycling | http://aims.fao.org/aos/agrovoc/c_12869 | https://www.eea.europa.eu/help/glossary/gemet-environmental-thesaurus/nutrient-cycle | https://www.wikidata.org/wiki/Q7070482 |
| SOIL FUNCTIONS | Agricultural production | | https://agrovoc.fao.org/browse/agrovoc/en/page/c_2ac7c9e1 | https://www.eea.europa.eu/help/glossary/gemet-environmental-thesaurus/agricultural-production | https://www.wikidata.org/wiki/Q56299222 |
| SOIL FUNCTIONS | Biomass production | | http://aims.fao.org/aos/agrovoc/c_adfffd29 | | https://www.wikidata.org/wiki/Q33059530 |
| SOIL FUNCTIONS | Soil fertility | | http://aims.fao.org/aos/agrovoc/c_7170 | https://www.eea.europa.eu/help/glossary/eea-glossary/soil-fertility | https://www.wikidata.org/wiki/Q202080 |
| SOIL FUNCTIONS | Disease and pest regulation | https://soilhealthbenchmarks.eu/glossary/disease-and-pest-regulation | | | |
| SOIL FUNCTIONS | Habitat provision | https://soilhealthbenchmarks.eu/glossary/habitat-provision | http://aims.fao.org/aos/agrovoc/c_3456 | | |
| SOIL FUNCTIONS | Biodiversity | | http://aims.fao.org/aos/agrovoc/c_33949 | https://www.eea.europa.eu/help/glossary/gemet-environmental-thesaurus/biodiversity | https://www.wikidata.org/wiki/Q47041 |
| SOIL FUNCTIONS | Soil processes | https://soilhealthbenchmarks.eu/glossary/soil-processes | | https://www.eea.europa.eu/help/glossary/gemet-environmental-thesaurus/soil-process | |
| SOIL FUNCTIONS | Soil properties | https://soilhealthbenchmarks.eu/glossary/soil-properties | http://aims.fao.org/aos/agrovoc/c_330883 | | |
| SOIL INDICATORS |  | https://soilhealthbenchmarks.eu/glossary/soil-indicators | https://agrovoc.fao.org/browse/agrovoc/en/page/c_24040 | | |
| SOIL INDICATORS | Soil health index | https://soilhealthbenchmarks.eu/glossary/soil-health-index | | | |
| SOIL INDICATORS | Soil descriptor | https://soilhealthbenchmarks.eu/glossary/soil-descriptor | | | |
| SOIL INDICATORS | Soil Health Indicators | https://soilhealthbenchmarks.eu/glossary/soil-health-indicators | | | |
| SOIL INDICATORS | Pollutant | https://soilhealthbenchmarks.eu/glossary/pollutant | http://aims.fao.org/aos/agrovoc/c_6076 | https://www.eea.europa.eu/help/glossary/gemet-environmental-thesaurus/pollutant | https://www.wikidata.org/wiki/Q19829510 |
| SOIL INDICATORS | Soil pollutant | | http://aims.fao.org/aos/agrovoc/c_e9c250f9 | | https://www.wikidata.org/wiki/Q50430163 |
| SOIL INDICATORS | Excess nutrients | https://soilhealthbenchmarks.eu/glossary/excess-nutrients | | | |
| SOIL INDICATORS | Salt-affected soil | https://soilhealthbenchmarks.eu/glossary/salt-affected-soil | | | https://www.wikidata.org/wiki/Q58387340 |
| SOIL INDICATORS | Soil organic carbon | https://soilhealthbenchmarks.eu/glossary/soil-organic-carbon | http://aims.fao.org/aos/agrovoc/c_389fe908 | https://www.eea.europa.eu/help/glossary/eea-glossary/soil-organic-carbon | https://www.wikidata.org/wiki/Q113009567 |
| SOIL INDICATORS | Soil organic carbon stock | https://soilhealthbenchmarks.eu/glossary/soil-organic-carbon-stock | | | https://www.wikidata.org/wiki/Q113009599 |
| SOIL INDICATORS | Soil structure including soil bulk density and absence of soil sealing and erosion | https://soilhealthbenchmarks.eu/glossary/soil-structure-including-soil-bulk-density-and-absence-of-soil-sealing-and-erosion | | | |
| SOIL INDICATORS | Soil structure | https://soilhealthbenchmarks.eu/glossary/soil-structure | http://aims.fao.org/aos/agrovoc/c_7196 | https://www.eea.europa.eu/help/glossary/gemet-environmental-thesaurus/soil-structure | https://www.wikidata.org/wiki/Q889539 |
| SOIL INDICATORS | Bulk density | https://soilhealthbenchmarks.eu/glossary/bulk-density | http://aims.fao.org/aos/agrovoc/c_7167 | | https://www.wikidata.org/wiki/Q899249 |
| SOIL INDICATORS | Soil sealing | https://soilhealthbenchmarks.eu/glossary/soil-sealing | http://aims.fao.org/aos/agrovoc/c_f9554e7c | https://www.eea.europa.eu/help/glossary/eea-glossary/soil-sealing | https://www.wikidata.org/wiki/Q3964127 |
| SOIL INDICATORS | Soil erosion | https://soilhealthbenchmarks.eu/glossary/soil-erosion | https://agrovoc.fao.org/browse/agrovoc/en/page/c_2651 | https://www.eea.europa.eu/help/glossary/eea-glossary/soil-erosion | https://www.wikidata.org/wiki/Q889503 |
| SOIL INDICATORS | Soil biodiversity | https://soilhealthbenchmarks.eu/glossary/soil-biodiversity | | | |
| SOIL INDICATORS | Biodiversity | https://soilhealthbenchmarks.eu/glossary/biodiversity | http://aims.fao.org/aos/agrovoc/c_33949 | | |
| SOIL INDICATORS | Soil nutrients | https://soilhealthbenchmarks.eu/glossary/soil-nutrients | | | |
| SOIL INDICATORS | pH | https://soilhealthbenchmarks.eu/glossary/ph | https://agrovoc.fao.org/browse/agrovoc/en/page/c_5753 | https://www.eionet.europa.eu/gemet/en/concept/14636 | https://www.wikidata.org/wiki/Q565649 |
| SOIL INDICATORS | Vegetation cover | https://soilhealthbenchmarks.eu/glossary/vegetation-cover | http://aims.fao.org/aos/agrovoc/c_25409 | https://www.eionet.europa.eu/gemet/en/concept/8908 | https://www.wikidata.org/wiki/Q105217314 |
| SOIL INDICATORS | Landscape heterogeneity | https://soilhealthbenchmarks.eu/glossary/landscape-heterogeneity | | | |
| SOIL INDICATORS | Forest cover | https://soilhealthbenchmarks.eu/glossary/forest-cover | http://aims.fao.org/aos/agrovoc/c_9000180 | | https://www.wikidata.org/wiki/Q4259364 |
| SOIL HEALTH |  | https://soilhealthbenchmarks.eu/glossary/soil-health | http://aims.fao.org/aos/agrovoc/c_a9645d28 | | https://www.wikidata.org/wiki/Q7554920 |
| SOIL HEALTH | Soil | https://soilhealthbenchmarks.eu/glossary/soil | http://aims.fao.org/aos/agrovoc/c_7156 | https://www.eionet.europa.eu/gemet/en/concept/7843 | https://www.wikidata.org/wiki/Q36133 |
| SOIL HEALTH | Soil health assessment | https://soilhealthbenchmarks.eu/glossary/soil-health-assessment | | | |
| SOIL HEALTH | Soil condition | https://soilhealthbenchmarks.eu/glossary/soil-condition | | | |
| SOIL HEALTH | Soil capability | https://soilhealthbenchmarks.eu/glossary/soil-capability | | | |
| SOIL HEALTH | Soil quality | https://soilhealthbenchmarks.eu/glossary/soil-quality | http://aims.fao.org/aos/agrovoc/c_a9645d28 | https://www.eea.europa.eu/help/glossary/gemet-environmental-thesaurus/soil-quality | https://www.wikidata.org/wiki/Q2034420 |
| SOIL HEALTH | Soil fertility | https://soilhealthbenchmarks.eu/glossary/soil-fertility | http://aims.fao.org/aos/agrovoc/c_7170 | | https://www.wikidata.org/wiki/Q202080 |
| SOIL HEALTH | Soil productivity | https://soilhealthbenchmarks.eu/glossary/soil-productivity | http://aims.fao.org/aos/agrovoc/c_7170 | | https://www.wikidata.org/wiki/Q115345770 |
| SOIL HEALTH | Soil security | https://soilhealthbenchmarks.eu/glossary/soil-security | http://aims.fao.org/aos/agrovoc/c_f42c0d0e | | |
| SOIL HEALTH | Soil resilience | https://soilhealthbenchmarks.eu/glossary/soil-resilience | | | https://www.wikidata.org/wiki/Q7554953 |
| SOIL HEALTH | Soil protection | https://soilhealthbenchmarks.eu/glossary/soil-protection | | | |
| SOIL HEALTH | Soil threat | https://soilhealthbenchmarks.eu/glossary/soil-threat | | | |
| SOIL HEALTH | Soil degradation | https://soilhealthbenchmarks.eu/glossary/soil-degradation | http://aims.fao.org/aos/agrovoc/c_7168 | |  |
| SOIL HEALTH | Soil contamination | https://soilhealthbenchmarks.eu/glossary/soil-contamination | | | https://www.wikidata.org/wiki/Q212734 |
| SOIL HEALTH | Contaminant | https://soilhealthbenchmarks.eu/glossary/contaminant | http://aims.fao.org/aos/agrovoc/c_4ba65a4a | | https://www.wikidata.org/wiki/Q2995424 |
| SOIL HEALTH | Contaminated site | https://soilhealthbenchmarks.eu/glossary/contaminated-site | | | https://www.wikidata.org/wiki/Q5164914 |
| SOIL HEALTH | Regeneration | https://soilhealthbenchmarks.eu/glossary/regeneration | | | https://www.wikidata.org/wiki/Q193119 |
| MANAGEMENT PRACTICES |  | https://soilhealthbenchmarks.eu/glossary/management-practices | | | |
| MANAGEMENT PRACTICES | Soil management practices | https://soilhealthbenchmarks.eu/glossary/soil-management-practices | | | |
| MANAGEMENT PRACTICES | Managed soils | https://soilhealthbenchmarks.eu/glossary/managed-soils | | | |
| MANAGEMENT PRACTICES | Nature-based solutions | https://soilhealthbenchmarks.eu/glossary/nature-based-solutions | http://aims.fao.org/aos/agrovoc/c_0fafbb81 | | https://www.wikidata.org/wiki/Q2421934 |
| MANAGEMENT PRACTICES | Management practices | https://soilhealthbenchmarks.eu/glossary/management-practices | | | |
| MANAGEMENT PRACTICES | Animal management | https://soilhealthbenchmarks.eu/glossary/animal-management | | https://www.eionet.europa.eu/gemet/en/concept/429 | https://www.wikidata.org/wiki/Q80962 |
| MANAGEMENT PRACTICES | Biodiversity management | https://soilhealthbenchmarks.eu/glossary/biodiversity-management | | | |
| MANAGEMENT PRACTICES | Carbon and nutrient management | https://soilhealthbenchmarks.eu/glossary/carbon-and-nutrient-management | | | |
| MANAGEMENT PRACTICES | Disturbance management | https://soilhealthbenchmarks.eu/glossary/disturbance-management | | | |
| MANAGEMENT PRACTICES | Grazing management | https://soilhealthbenchmarks.eu/glossary/grazing-management | http://aims.fao.org/aos/agrovoc/c_3367 | | https://www.wikidata.org/wiki/Q113207774 |
| MANAGEMENT PRACTICES | Pest management | https://soilhealthbenchmarks.eu/glossary/pest-management | http://aims.fao.org/aos/agrovoc/c_13262 | https://www.eionet.europa.eu/gemet/en/concept/6139 | https://www.wikidata.org/wiki/Q2306047 |
| MANAGEMENT PRACTICES | Pollutant management | https://soilhealthbenchmarks.eu/glossary/pollutant-management | | | |
| MANAGEMENT PRACTICES | Recreational use management | https://soilhealthbenchmarks.eu/glossary/recreational-use-management | | | |
| MANAGEMENT PRACTICES | Soil management | https://soilhealthbenchmarks.eu/glossary/soil-management | http://aims.fao.org/aos/agrovoc/c_7176 | | https://www.wikidata.org/wiki/Q7554944 |
| MANAGEMENT PRACTICES | Terrain management | https://soilhealthbenchmarks.eu/glossary/terrain-management | | | |
| MANAGEMENT PRACTICES | Vegetation management | https://soilhealthbenchmarks.eu/glossary/vegetation-management | | | https://www.wikidata.org/wiki/Q2055641 |
| MANAGEMENT PRACTICES | Water management | https://soilhealthbenchmarks.eu/glossary/water-management | http://aims.fao.org/aos/agrovoc/c_8320 | | https://www.wikidata.org/wiki/Q1501619 |
| LAND USES |  | https://soilhealthbenchmarks.eu/glossary/land-uses | https://agrovoc.fao.org/browse/agrovoc/en/page/c_4182 | | |
| LAND USES | Land-use change | https://soilhealthbenchmarks.eu/glossary/land-use-change | http://aims.fao.org/aos/agrovoc/c_fac4b794 | | |
| LAND USES | Land cover | https://soilhealthbenchmarks.eu/glossary/land-cover | http://aims.fao.org/aos/agrovoc/c_37897 | | |
| LAND USES | Land | https://soilhealthbenchmarks.eu/glossary/land | http://aims.fao.org/aos/agrovoc/c_4172 | | |
| LAND USES | Soil district | https://soilhealthbenchmarks.eu/glossary/soil-district | | | |
| LAND USES | Urban system | https://soilhealthbenchmarks.eu/glossary/urban-system | | | |
| LAND USES | Urban agriculture | https://soilhealthbenchmarks.eu/glossary/urban-agriculture | http://aims.fao.org/aos/agrovoc/c_35707 | | |
| LAND USES | Urban parks | https://soilhealthbenchmarks.eu/glossary/urban-parks | | | |
| LAND USES | Urban green infrastructures | https://soilhealthbenchmarks.eu/glossary/urban-green-infrastructures | | | |
| LAND USES | Blue-green infrastructure | https://soilhealthbenchmarks.eu/glossary/blue-green-infrastructure | | | |
| LAND USES | Peri-urban forests | https://soilhealthbenchmarks.eu/glossary/peri-urban-forests | | | |
| LAND USES | Urban forestry | https://soilhealthbenchmarks.eu/glossary/urban-forestry | http://aims.fao.org/aos/agrovoc/c_32620 | | |
| LAND USES | Allotments | https://soilhealthbenchmarks.eu/glossary/allotments | http://aims.fao.org/aos/agrovoc/c_294 | | |
| LAND USES | Green corridors | https://soilhealthbenchmarks.eu/glossary/green-corridors | | | |
| LAND USES | Gardens | https://soilhealthbenchmarks.eu/glossary/gardens | http://aims.fao.org/aos/agrovoc/c_3196 | | |
| LAND USES | Urban urbanized areas | https://soilhealthbenchmarks.eu/glossary/urban-urbanized-areas | http://aims.fao.org/aos/agrovoc/c_8085 | | |
| LAND USES | Built-up areas | https://soilhealthbenchmarks.eu/glossary/built-up-areas | | | |
| LAND USES | Urban soils | https://soilhealthbenchmarks.eu/glossary/urban-soils | | | |
| LAND USES | Technosols (including constructed Technosols) | https://soilhealthbenchmarks.eu/glossary/technosols-including-constructed-technosols | http://aims.fao.org/aos/agrovoc/c_3a0750ba | | |
| LAND USES | Artefacts | https://soilhealthbenchmarks.eu/glossary/artefacts | | | |
| LAND USES | SUITMA | https://soilhealthbenchmarks.eu/glossary/suitma | | | |
| LAND USES | Forest | https://soilhealthbenchmarks.eu/glossary/forest | http://aims.fao.org/aos/agrovoc/c_3062 | | |
| LAND USES | Forest expansion | https://soilhealthbenchmarks.eu/glossary/forest-expansion | | | |
| LAND USES | Deforestation | https://soilhealthbenchmarks.eu/glossary/deforestation | http://aims.fao.org/aos/agrovoc/c_15590 | | |
| LAND USES | Reforestation | https://soilhealthbenchmarks.eu/glossary/reforestation | http://aims.fao.org/aos/agrovoc/c_13802 | | |
| LAND USES | Above-Ground (Vegetation) Biomass | https://soilhealthbenchmarks.eu/glossary/above-ground-vegetation-biomass | http://aims.fao.org/aos/agrovoc/c_1373987580598 | | |
| LAND USES | Below-Ground Biomass (Root biomass) | https://soilhealthbenchmarks.eu/glossary/below-ground-biomass-root-biomass | http://aims.fao.org/aos/agrovoc/c_363a2055 | | |
| LAND USES | Production Forest | https://soilhealthbenchmarks.eu/glossary/production-forest | http://aims.fao.org/aos/agrovoc/c_28123 | | |
| LAND USES | Protection forest | https://soilhealthbenchmarks.eu/glossary/protection-forest | http://aims.fao.org/aos/agrovoc/c_28127 | | |
| LAND USES | Multiple use forest | https://soilhealthbenchmarks.eu/glossary/multiple-use-forest | http://aims.fao.org/aos/agrovoc/c_24874 | | |
| LAND USES | Forest ownership | https://soilhealthbenchmarks.eu/glossary/forest-ownership | | | |
| LAND USES | Forest private ownership | https://soilhealthbenchmarks.eu/glossary/forest-private-ownership | | | |
| LAND USES | Forest public ownership | https://soilhealthbenchmarks.eu/glossary/forest-public-ownership | | | |
| LAND USES | Afforestation | https://soilhealthbenchmarks.eu/glossary/afforestation | http://aims.fao.org/aos/agrovoc/c_162 | | |
| LAND USES | Agriculture | https://soilhealthbenchmarks.eu/glossary/agriculture | http://aims.fao.org/aos/agrovoc/c_203 | | |
| LAND USES | Multi-species system | https://soilhealthbenchmarks.eu/glossary/multi-species-system | | | |
| LAND USES | Arable land | https://soilhealthbenchmarks.eu/glossary/arable-land | http://aims.fao.org/aos/agrovoc/c_18948 | | |
| LAND USES | Grassland | https://soilhealthbenchmarks.eu/glossary/grassland | http://aims.fao.org/aos/agrovoc/c_3366 | | |
| LAND USES | Fallow | https://soilhealthbenchmarks.eu/glossary/fallow | http://aims.fao.org/aos/agrovoc/c_34007 | | |
| LAND USES | Agroforestry | https://soilhealthbenchmarks.eu/glossary/agroforestry | http://aims.fao.org/aos/agrovoc/c_207 | | |
| LAND USES | Monocrop | https://soilhealthbenchmarks.eu/glossary/monocrop | http://aims.fao.org/aos/agrovoc/c_4915 | | |
| LAND USES | Monoculture | https://soilhealthbenchmarks.eu/glossary/monoculture | http://aims.fao.org/aos/agrovoc/c_4915 | | |
| LAND USES | STAKEHOLDERS | https://soilhealthbenchmarks.eu/glossary/stakeholders | http://aims.fao.org/aos/agrovoc/c_9001020 | | |
| LAND USES | Decision Makers | https://soilhealthbenchmarks.eu/glossary/decision-makers | | | |
| LAND USES | Soil Regulator | https://soilhealthbenchmarks.eu/glossary/soil-regulator | | | |
| LAND USES | Soil Stewards | https://soilhealthbenchmarks.eu/glossary/soil-stewards | | | |
| LAND USES | Soil Health Advocates | https://soilhealthbenchmarks.eu/glossary/soil-health-advocates | | | |
| LAND USES | Land Stewards | https://soilhealthbenchmarks.eu/glossary/land-stewards | | | |
| LAND USES | Cooperatives | https://soilhealthbenchmarks.eu/glossary/cooperatives | http://aims.fao.org/aos/agrovoc/c_10090 | | |
| LAND USES | Citizens | https://soilhealthbenchmarks.eu/glossary/citizens | | | |
| LAND USES | Public Actors | https://soilhealthbenchmarks.eu/glossary/public-actors | | | |
| LAND USES | Supply Chain Actors (or private actors) | https://soilhealthbenchmarks.eu/glossary/supply-chain-actors-or-private-actors | | | |
| LAND USES | Initiatives and NGOs | https://soilhealthbenchmarks.eu/glossary/initiatives-and-ngos | | | |
| LAND USES | Initiatives | | http://aims.fao.org/aos/agrovoc/c_50239 | | |
| LAND USES | NGO | | http://aims.fao.org/aos/agrovoc/c_24068 | | |
| LAND USES | Knowledge and Education Actors | https://soilhealthbenchmarks.eu/glossary/knowledge-and-education-actors | | | |
| LAND USES | Multi-stakeholder Process (MSP) | https://soilhealthbenchmarks.eu/glossary/multi-stakeholder-process-msp | http://aims.fao.org/aos/agrovoc/c_5340b5c1 | | |
| LAND USES | Measurement Reporting and Verification (MRV) | https://soilhealthbenchmarks.eu/glossary/measurement-reporting-and-verification-mrv | | | |
| LAND USES | MRV Actors | https://soilhealthbenchmarks.eu/glossary/mrv-actors | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) |  | https://soilhealthbenchmarks.eu/glossary/integrated-soil-health-monitoring-framework-ishmf | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Monitoring metrics | https://soilhealthbenchmarks.eu/glossary/monitoring-metrics | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Practice metrics | https://soilhealthbenchmarks.eu/glossary/practice-metrics | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Result metrics | https://soilhealthbenchmarks.eu/glossary/result-metrics | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Outcome metrics | https://soilhealthbenchmarks.eu/glossary/outcome-metrics | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Indicator measurements | https://soilhealthbenchmarks.eu/glossary/indicator-measurements | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Sample | https://soilhealthbenchmarks.eu/glossary/sample | http://aims.fao.org/aos/agrovoc/c_875b7e95 | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Space | https://soilhealthbenchmarks.eu/glossary/space | http://aims.fao.org/aos/agrovoc/c_330769 | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Stats | https://soilhealthbenchmarks.eu/glossary/stats | http://aims.fao.org/aos/agrovoc/c_49978 | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Regions | https://soilhealthbenchmarks.eu/glossary/regions | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Landscapes | https://soilhealthbenchmarks.eu/glossary/landscapes | http://aims.fao.org/aos/agrovoc/c_4185 | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Local | https://soilhealthbenchmarks.eu/glossary/local | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Lighthouses | https://soilhealthbenchmarks.eu/glossary/lighthouses | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Lighthouse farms | https://soilhealthbenchmarks.eu/glossary/lighthouse-farms | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Citizen science | https://soilhealthbenchmarks.eu/glossary/citizen-science | http://aims.fao.org/aos/agrovoc/c_f888df97 | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Threshold | https://soilhealthbenchmarks.eu/glossary/threshold | | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Benchmark | https://soilhealthbenchmarks.eu/glossary/benchmark | http://aims.fao.org/aos/agrovoc/c_54db980a | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Parameter | https://soilhealthbenchmarks.eu/glossary/parameter | http://aims.fao.org/aos/agrovoc/c_7dfaf103 | | |
| Integrated Soil Health Monitoring Framework (ISHMF) | Weighting factor | https://soilhealthbenchmarks.eu/glossary/weighting-factor | | | |

**Comment by pvgenuchten** [comment]
Notice that the [benchmarks glossary](https://soilhealthbenchmarks.eu/glossary/) does not include a section on [soil threats](https://www.isqaper-is.eu/key-words/soil-threats). However all the known threats occur in other sections, such as indicators, but then without their negative connotation. For example "carbon stock" vs "decline in carbon stock". I wonder if soilwise supports this approach of listing threats in their neutral form under indicators. Or should soilwise use a dedicated codelist of soil threat keywords to be assigned to metadata records.

**Reply by chlebas** [reply]
soil threats can be an entry. In Benchmarks, they are more focused in ecosystem services than soil threats. Concerning your example, there is a difference between the indicator and the soil threat. The indicator is a measurement of a soil property, the soil threat is a variation of this indicator in time showing a decrease of the indicator value.

**Reply by wbcbugfree** [reply]
I agree. In the EEA's soil monitoring report, for example, SOC is an observable soil property and SOC loss is a soil threat indicator.

**Reply by pvgenuchten** [reply]
I discussed the topic of soil property vs soil indicator with Dries, for him the soil property alone is not an indicator, the indicator sometimes involves multiple soil properties and for the indicator some classes are identified. So an indicator always includes a classification

**Comment by chlebas** [comment]
I propose to the persons in charge of the glossary taru.sanden to put it in SKOS but it is a work to be done.

**Comment by wbcbugfree** [comment]
This glossary looks like a very clear two-tier structure, which means that if we import SKOS, we would use the "concept scheme"?

**Comment by cdonmez16** [comment]
Hi Paul, 
I think both approaches are possible. As far as I know, BENCHMARKS primarily focuses on soil functions, while the EUSO Soil Health dashboard, for instance, centers on soil threats. In Soilwise, we could potentially use either approach, though there might be a more common emphasis on soil threats. @pvgenuchten
